src/sequence/seq_crdt.rs

```diff
@@ -278,6 +278,8 @@ where
                     Error::Unexpected("The data is an unexpected inconsistent state".to_string())
                 })?;
 
+                // FIXME: Check that we actually have perms to be adding a new policy here, based on prev one...
```

Comment:
We've done permissions check before calling any of this struct's functions: https://github.com/maidsafe/sn_data_types/blob/master/src/sequence/mod.rs#L264-L269, this struct/internal-layer doesn't know about permissions. Is there a scenario wee are not covering yet?